### PR TITLE
Make the homepage flow better

### PR DESCRIPTION
**Pull request by cfcosta**
I have been thinking on how to make the homepage look a little better, everything to me seems out of place, so instead of arguing over how it should be, I decided to test it, and that's what I came up.

Changes:
  * The font to a more pleasant one (at least in Linux, Poppins is not properly antialiased).
  * Change the alignment and make better use of screen space.

Before:

After:

### CODE REVIEW

File: themes/kiali/static/css/kiali-home.css [hunk 1]
```diff
@@ -40,22 +40,24 @@ section#hero p.buttons {
   margin-top: var(--kiali-blue-spacing);
 }
 
-@media only screen and (min-width: 850px) {
-  section#hero {
-    padding-top: var(--kiali-bright-green-spacing);
-    padding-bottom: var(--kiali-bright-green-spacing);
-  }
-}
-
```
Comment (israel-hdez) [inline themes/kiali/static/css/kiali-home.css]: I'm not sure I agree with removing these paddings. It doesn't look bad, though.

File: themes/kiali/static/css/kiali-home.css [hunk 2]
```diff
@@ -40,22 +40,24 @@ section#hero p.buttons {
...
   section#hero p.buttons a.button {
     display: inline-block;
     margin-bottom: 0;
   }
 }
 
-
 /* "About" block styles */
+#about {
+  overflow: hidden;
```
Comment (israel-hdez) [inline themes/kiali/static/css/kiali-home.css]: Why do you need the hidden?
I'm not sure this is OK.

File: themes/kiali/static/css/kiali-home.css [hunk 3]
```diff
@@ -40,22 +40,24 @@ section#hero p.buttons {
...
+}
+
+#about-description {
+  max-width: 600px;
+}
+#intro-video {
+  float: right;
```
Comment (israel-hdez) [inline themes/kiali/static/css/kiali-home.css]: The video cannot float at all resolutions. It needs to fallback to be placed at the bottom of the about text on small width.
It doesn't look good if you visit the website on a phone in landscape orientation. It also looks bad if you have two browser windows side by side -- you may want to see Kiali and Istio websites at the same time ;)

Comment (israel-hdez) [reply themes/kiali/static/css/kiali-home.css]: You also may need to add margins to the left of the video. On not too wide screens the text is too close to the video:
image

File: themes/kiali/static/css/kiali.css [hunk 4]
```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Poppins:300,400,500&display=swap');
+@import url("https://fonts.googleapis.com/css?family=Fira+Sans+Condensed&display=swap");
```
Comment (israel-hdez) [inline themes/kiali/static/css/kiali.css]: Please, specify the '300,400,500' weights, and add the other weights you used.
There are some bolder and thinners texts that are lost.

For example the text "Kiali answers the questions:" is no longer rendering bolder than the listed questions.

File: themes/kiali/static/css/kiali-home.css [hunk 5]
```diff
@@ -76,16 +78,14 @@ div#features-blocks a:hover div {
 }
 
 div#features-blocks div {
-  height: 210px;
-  width: 290px;
+  height: 150px;
+  width: 420px;
```
Comment (israel-hdez) [inline themes/kiali/static/css/kiali-home.css]: These new width are not mobile friendly.
If you want this width for desktop it's OK. But for mobile screens the feature cards need to adapt to the width of the screen.

File: themes/kiali/layouts/partials/home/features-section.html [hunk 6]
```diff
@@ -4,57 +4,54 @@ <h1>Features</h1>
...
           <h2><img src="img/configuration.svg" alt="" />Configuration</h2>
-          <p>See and edit Istio<br />CRD YAML</p>
+          <p>See and edit Istio CRD YAML</p>
         </div>
       </a>
-
-      <div class="more-coming">
-        <p><img src="img/more.svg" alt="" /></p>
-        <h2>More to come...</h2>
-      </div>
```
Comment (israel-hdez) [inline themes/kiali/layouts/partials/home/features-section.html]: :+1:

**Comment by jshaughn** [comment]
@cfcosta This PR should be tied to a github issue and a sprint project.  Remember to prioritize the project backlog, especially early in a sprint.

**Comment by cfcosta** [comment]
Closing, will create a new PR on the design repo later in the sprint.

**Pull request closed without merging**